templates/vhost.conf

```diff
@@ -30,4 +30,12 @@
         CustomLog logs/{{ _website_domain }}_access.log combined
 {% endif %}
 
+{% if wsgi_script is defined and wsgi_script %}
+{% if not _force_tls or port|int == 443 %}
```

Comment:
Why not drop that as a file in {{ _vhost_confdir }}/ ? (cause this will deal with _not_force_tls directly there) ?